qiskit_experiments/library/randomized_benchmarking/layer_fidelity.py

```diff
@@ -454,16 +482,21 @@ def __circuit_body(
                 sample = rng.integers(NUM_1Q_CLIFFORD)
                 cliffs_1q[k] = compose_1q(cliffs_1q[k], sample)
                 circ._append(_to_gate_1q(sample), (circ.qubits[q],), ())
-            circ._append(barrier_inst)
+            for barrier_inst in barrier_inst_lst:
+                circ._append(barrier_inst)
             # add two qubit gates
             for j, qpair in enumerate(two_qubit_layer):
                 circ._append(gate2q, tuple(circ.qubits[q] for q in qpair), ())
                 cliffs_2q[j] = compose_2q(cliffs_2q[j], gate2q_cliff)
                 # TODO: add dd if necessary
             for k, q in enumerate(one_qubits):
                 # TODO: add dd if necessary
-                pass
-            circ._append(barrier_inst)
+                #if there is a min_delay, just need
+                #to add to one of the qubits 
+                if min_delay is not None and k==0:
```

Comment:
What if you have `physical_qubits=[0, 1, 2, 3]` and `two_qubit_layers=[[(0, 1), (2, 3)]]`? Won't `one_qubits` be empty and `min_delay` silently ignored?

Reply:
Correct. I added something to the parameter comment to note this unless we thing there should be a warning?